src/components/Widgets/Widget.style.ts

```diff
@@ -19,9 +21,14 @@ export const WidgetWrapper = styled(Box, {
     },
   }),
   zIndex: 2,
-  // [theme.breakpoints.up('sm' as Breakpoint)]: {
-  //   maxWidth: 416,
-  // },
+
+  [theme.breakpoints.up('sm' as Breakpoint)]: {
+    ...(!isExpanded && {
```

Comment:
i think it is quite difficult to read that part, maybe that could be refactored like:

```
export const WidgetWrapper = styled(Box, {
  shouldForwardProp: (prop) => !['welcomeScreenClosed', 'isExpanded'].includes(prop),
})<WidgetWrapperProps>(({ theme, welcomeScreenClosed, isExpanded }) => {
  const baseStyle = {
    width: '100%',
    position: 'relative',
    margin: theme.spacing(0, 'auto'),
    zIndex: 2,
  };

  const hoverEffect = !welcomeScreenClosed ? {
    '&:hover': {
      marginTop: 0,
    }
  } : {};

  const responsiveHeight = theme.breakpoints.up('sm') as Breakpoint;
  const expandedStyle = !isExpanded ? {
    '& > div': {
      height: 616,
    }
  } : {};

  return {
    ...baseStyle,
    ...hoverEffect,
    [responsiveHeight]: {
      ...expandedStyle
    }
  };
});
```

Reply:
Thanks for the input @admsk2 !

Maybe we can have this discussion with @oktapodia and tcheee. 
Personally, I do not find the approach you suggested more readable but we should decide within our team what we see as best practise for such "more sophisticated" styled-components. Wdyt?